tutor/src/models/reference-book/node.js

```diff
@@ -163,7 +163,8 @@ class ReferenceBookNode extends BaseModel {
     if(!this.chapter_section || !this.chapter_section.chapter) {
       return this.titleText;
     }
-    return `${this.chapter_section.chapter}. ${this.titleText}`;
+    const chapterSection = this.chapter_section;
+    return `${chapterSection.chapter}.${chapterSection.section || ' '} ${this.titleText}`;
   }
```

Comment:
I know this isn't new, but chapter section has a format method that is better to use, it handles the case where section may be missing when it's 0. https://github.com/openstax/tutor-js/blob/main/tutor/src/models/chapter-section.js#L18-L23